packages/@react-aria/focus/test/FocusScope.test.js

```diff
@@ -331,6 +331,100 @@ describe('FocusScope', function () {
       expect(document.activeElement).toBe(outside);
     });
 
+    it('should restore focus to the previously focused node after a child with autoFocus unmounts', function () {
+      function Test({show}) {
+        return (
+          <div>
+            <input data-testid="outside" />
+            {show &&
+              <FocusScope restoreFocus>
+                <input data-testid="input1" />
+                <input data-testid="input2" autoFocus />
+                <input data-testid="input3" />
+              </FocusScope>
+            }
+          </div>
+        );
+      }
+
+      let {getByTestId, rerender} = render(<Test />);
+
+      let outside = getByTestId('outside');
+      act(() => {outside.focus();});
+
+      rerender(<Test show />);
+
+      let input2 = getByTestId('input2');
+      expect(document.activeElement).toBe(input2);
+
+      rerender(<Test />);
+
+      expect(document.activeElement).toBe(outside);
+    });
+
+    it('should restore focus to the previously focused node when tabbing away from a child with autoFocus', function () {
```

Comment:
based on this description I'd expect the final check on active element to be "outside" not "after".

I feel like all three of these tests should check the activeElement before adding the <FocusScope /> and checking that the focus is within it since they all say "restore focus".

Reply:
Yes the phrasing here is off, I'll fix the description. In this case focus is "moving after" rather than "restoring".

And just so I understand you (I think you're missing a word), in this case are you suggesting to check if `activeElement` is `outside` before `rerender(<Test show />)`?